src/uiTest/scala/com/virtuslab/gitmachete/uitest/UIScenarioSuite.scala

```diff
@@ -0,0 +1,104 @@
+package com.virtuslab.gitmachete.uitest
+
+import com.virtuslab.gitmachete.testcommon.BaseGitRepositoryBackedIntegrationTestSuite
+import com.virtuslab.gitmachete.testcommon.SetupScripts.SETUP_README_SCENARIOS
+import org.junit._
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.virtuslab.ideprobe.ProbeDriver
+
+object UIScenarioSuite extends UISuite {}
+
+
+@RunWith(classOf[JUnit4])
+class UIScenarioSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_README_SCENARIOS) {
+  import UIScenarioSuite._
+
+  private val project = intelliJ.project
+
+  private val probe: ProbeDriver = intelliJ.probe
+
+  @Before
+  def beforeEach(): Unit = {
+    probe.openProject(rootDirectoryPath)
+    project.configure()
+    probe.await()
+    project.openGitMacheteTab()
+    project.toggleListingCommits()
+    intelliJ.ide.findAndResizeIdeFrame()
+  }
+
+  @After
+  def afterEach(): Unit = {
```

Comment:
Nit: maybe extract the method to some shared place? like `UISuite`

Reply:
Actually, extract the **logic** as a `@After` probably won't take effect if it's on a inherited method :/